### PR TITLE
Fix the review comments for EA3 release.

### PR DESCRIPTION
**Pull request by shaiknayabrasool-s**

### CODE REVIEW

File: docs/reference/Func/SECValidateAccessToken.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Input :
 
 dwFlags  -  Modify behavior from defaults. See fJWT_validate_* below.
 
-*vpOptionalParams  -  pszCustomClaimName.
+*vpOptionalParams  -  pszCustomClaimName value can be filled with using this structure. Please refer this [JWT_VALIDATE_OPTIONAL_PARAMS.md](/domino-c-api-docs/reference/Data/JWT_VALIDATE_OPTIONAL_PARAMS.md).
 
-dwMaxEmailSize  -  ResourceCallback.
+dwMaxEmailSize  -  MaxEmailSize.
```
Comment (saravananbalakris) [inline docs/reference/Func/SECValidateAccessToken.md]: dwMaxEmailSize  -  Maximum length of email buffer size.

File: docs/reference/Func/SECValidateAccessToken.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ Input :
 
 dwFlags  -  Modify behavior from defaults. See fJWT_validate_* below.
 
-*vpOptionalParams  -  pszCustomClaimName.
+*vpOptionalParams  -  pszCustomClaimName value can be filled with using this structure. Please refer this [JWT_VALIDATE_OPTIONAL_PARAMS.md](/domino-c-api-docs/reference/Data/JWT_VALIDATE_OPTIONAL_PARAMS.md).
```
Comment (saravananbalakris) [inline docs/reference/Func/SECValidateAccessToken.md]: *vpOptionalParams  -  Value can be filled with using structure JWT_VALIDATE_OPTIONAL_PARAMS. Please refer this [JWT_VALIDATE_OPTIONAL_PARAMS.md](/domino-c-api-docs/reference/Data/JWT_VALIDATE_OPTIONAL_PARAMS.md).

**Review by saravananbalakris — CHANGES_REQUESTED** [review]
Please incorporate changes

File: docs/reference/Func/SECValidateAccessToken.md (hunk 1 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/SECValidateAccessToken.md]: Modified

File: docs/reference/Func/SECValidateAccessToken.md (hunk 2 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/SECValidateAccessToken.md]: Corrected.

**Review by saravananbalakris — APPROVED** [review]
Looks fine

**Review by raghu-madat — APPROVED** [review]

**Pull request merged**